src/game.ts

```diff
@@ -692,6 +694,31 @@ function drawBoardBackground(
   );
 }
 
+function drawRankAndFileLabels(
+  ctx: CanvasRenderingContext2D,
+  rect: { x: number; y: number; width: number; height: number },
+  ranks: string[],
+  files: string[],
+) {
+  const boxSize = (rect.width / GRID_DIM);
+  const fontSize = Math.round(boxSize / 5);
```

Comment:
not sure the best way to handle scaling. so I cheated a bit and just based my size values on alread-scaled GRID_DIM numbers